docs/user_guide/overview.rst

```diff
@@ -117,6 +119,28 @@ participants in project.yml) specified when generating the startup kits in the p
 correct IP. If the FL server is on an internal network without a DNS hostname, in Ubuntu, an entry may need to be added
 to ``/etc/hosts`` with the internal IP and the hostname.
 
+Federated learning servers ($SERVER_NAME.zip)
+=============================================
+Server will coordinate the federated learning training and be the main hub all clients and admin
+clients connect to.
+
+After unzipping the package server.zip, run the start.sh file from the "startup" folder you unzipped to start the server.
+
+The rootCA.pem file is pointed to by "ssl_root_cert" in fed_server.json.  If you plan to move/copy it to a different place,
+you will need to modify fed_server.json.  The same applies to the other two files, server.crt and server.key.
+
+.. note::
+
+   When launching the FL server inside a docker with ``docker run``, use ``--net=host`` to map hostname into that
+   docker instance.  For secure gRPC communication, the FL server has to bind to the hostname specified in the
+   provisioning stage. Always make sure that hostname is what FL server can bind to. Additionally,
+   the port that the server communicates on must also not be blocked by any firewalls.
+
+If clients from other machines cannot connect to the server, make sure that the hostname (name of the server under
+participants in project.yml) specified when generating the startup kits in the provisioning process resolves to the
+correct IP. If the FL server is on an internal network without a DNS hostname, in Ubuntu, an entry may need to be added
+to ``/etc/hosts`` with the internal IP and the hostname.
```

Comment:
Since both overseer and FL server need this section, should we just have them in one place and add link to ref/redirect to that section?